File: Archive/fizzbuzz/python/fizzbuzz-array.py
output = []
for n in range(1, 2000001):
    if (n) % 3 == 0 and (n) % 5 == 0:
        output.append("fizzbuzz")
    elif (n) % 3 == 0:
        output.append("fizz")
    elif (n) % 5 == 0:
        output.append("buzz")
    else:
        output.append(f"{n}")

print(output)

File: Archive/fizzbuzz/golang/fizzbuzz-array.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var s []string
	for i := 1; i < 2000001; i++ {
		if i%3 == 0 && i%5 == 0 {
			s = append(s, "fizzbuzz \n")
		} else if i%3 == 0 {
			s = append(s, "fizz \n")
		} else if i%5 == 0 {
			s = append(s, "buzz \n")
		} else {
			n := strconv.Itoa(i)
			o := fmt.Sprintf("%s \n", n)
			s = append(s, o)
		}
	}
	fmt.Println(s)
}
